srcJava/com/rti/perftest/ddsimpl/RTIDDSImpl.java

```diff
@@ -575,19 +572,9 @@ public IMessagingReader createReader(String topicName,
         TopicDescription  topic_desc = topic; // Used to create the DDS DataReader
 
         String qosProfile;
-        if (THROUGHPUT_TOPIC_NAME.VALUE.equals(topicName)) {
-            qosProfile = "ThroughputQos";
-        } else if (LATENCY_TOPIC_NAME.VALUE.equals(topicName)) {
-            qosProfile = "LatencyQos";
-        } else if (ANNOUNCEMENT_TOPIC_NAME.VALUE.equals(topicName)) {
-            qosProfile = "AnnouncementQos";
-        }
-        else {
-            System.err.println(
-                    "topic name must either be " +
-                    THROUGHPUT_TOPIC_NAME.VALUE + " or " +
-                    LATENCY_TOPIC_NAME.VALUE  + " or " +
-                    ANNOUNCEMENT_TOPIC_NAME.VALUE);
+        qosProfile = getQoSProfileName(topicName);
```

Comment:
remember that here the Code Convention I believe reminds the same (check)

Reply:
Yes, it's the same, according to the general java code convention.